sdk/rust/oak_derive/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "oak_derive"
+version = "0.1.0"
+authors = ["Daan de Graaf <daandegraaf9@gmail.com>"]
```

Comment:
[Use](https://opensource.google/docs/github/accounts/#config) an `@google.com` address?  (which I guess also applies to git commit metadata)